bee-node/Dockerfile

```diff
@@ -0,0 +1,58 @@
+############################
+# Prepare dependencies recipe
+############################
+FROM lukemathwalker/cargo-chef as planner
+
+WORKDIR /src
+COPY . .
```

Comment:
Are you sure this is going to work? The `WORKDIR` points to a directory that doesn't exist at the target, because to copy to `.`.
```
COPY . /bee
WORKDIR /bee
```
or just:
```
COPY . .
```

Reply:
Yes, this works; Docker takes care of creating the WORKDIR.